tools/jumpstart

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# Download pre-built webpack for current git SHA from GitHub
```

Comment:
I know you like that name, but can we pretty please name it to something descriptive? download-dist, or maybe cached-webpack? (dist seems more appropriate, as that's the directory it uses).

This current split between make_dist.py and this shell script bothers me -- the remaining Python script does not do anything any more that is worth writing in Python. Is there any use case for even having two separate scripts? I'd rather just completely port make_dist to a shell script.

Reply:
I'm trying to capture the essence of "get this worktree quickly up and running and ready for new hacking".  I don't actually care what's happening to make that happen.  I just want the first time I type "make" to be fast.

I actually also consider to fetch the bots and/or call `./autogen.sh` for you (possibly using `"$@"` for extra arguments).

Reply:
Let's chat about this on IRC :)